src/AutoRest.CSharp/AutoRest.CSharp.csproj

```diff
@@ -21,7 +21,7 @@
     <PackageReference Include="Azure.Core" Version="1.26.0" />
     <PackageReference Include="Azure.Core.Expressions.DataFactory" Version="1.0.0-alpha.20221110.1" />
     <PackageReference Include="Azure.Core.Experimental" Version="0.1.0-preview.18" />
-    <PackageReference Include="Azure.ResourceManager" Version="1.3.1" />
+    <PackageReference Include="Azure.ResourceManager" Version="1.4.0-alpha.20221218.1" />
```

Comment:
We will update to a new stable version once we have that.
We need the new version because we need the model factory in resource manager for some manual-written models (for instance SubResource)

Reply:
1.4.0 just releases. updated to 1.4.0